kettle-plugins/formats/src/main/java/org/pentaho/big/data/kettle/plugins/formats/impl/parquet/output/ParquetOutput.java

```diff
@@ -171,4 +197,25 @@ public boolean init( StepMetaInterface smi, StepDataInterface sdi ) {
     }
     return true;
   }
+
+  private void copyFileAndDeleteTempSource( ) throws KettleFileException, IOException {
```

Comment:
Does the Snowflake VFS not already delete the temp file on completion?  If not, seems like it should-- I wouldn't think each consumer of the vfs impl should be responsible.

Reply:
Yes, it was not implemented at all other than to throw an unsupported operation